### PR TITLE
some locale work and react work for simple paypal page

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: components/simple-paypal.jsx [hunk 1]
```diff
@@ -1,44 +1,68 @@
...
+        {message}
+      </button>
+    );
+  }
+});
 
 var simplePaypal = React.createClass({
+  mixins: [IntlMixin],
+  getInitialState: function() {
+    var value = "";
```
Comment (alicoding) [inline components/simple-paypal.jsx]: I don't think you need to do this?

File: components/simple-paypal.jsx [hunk 2]
```diff
@@ -1,44 +1,68 @@
...
-      updateDonateButtonText(amountSelected);
-    }
-
     // check if a pre-selected amount is passed via the URL
     if (window.location.hash.match(/#amount-\d+?/)) {
-      var preSelected = window.location.hash.substring(8);
-      $theForm.find('[name="donation_amount_other"]').val(preSelected);
-      updateDonateButtonText(preSelected);
+      this.setState({
+        value: window.location.hash.substring(8)
```
Comment (alicoding) [inline components/simple-paypal.jsx]: Preferably to use what react router expose here but its fine for now

File: pages/paypal-donate.jsx [hunk 3]
```diff
@@ -2,8 +2,10 @@ var React = require('react');
 var Footer = require('../components/footer.jsx');
```
Comment (alicoding) [inline pages/paypal-donate.jsx]: Do you mind changing all these to be es6 so its consistent?

**Comment by alicoding** [comment]
R+

File: components/simple-paypal.jsx (hunk 2 above)
Comment (ScottDowne) [inline components/simple-paypal.jsx]: I tried and wasn't super happy with it.

It requires more work.

```
import Router from 'react-router';
console.log(Router.HashLocation.getCurrentPath())
```
vs
```
console.log(window.location.hash);
```

Not only that, but `Router.HashLocation.getCurrentPath` doesn't work server side, it fails on getting `window.location.hash`...

Feels super redundant to me.

**Pull request merged**